src/util/postgres_client.js

```diff
@@ -411,7 +448,16 @@ class PostgresTable {
         const { schema, name, db_indexes, client, init_function } = table_params;
         this.name = name;
         this.init_function = init_function;
-        this.db_indexes = db_indexes;
+        this.db_indexes = _.concat([{
+                fields: {
+                    _id: 1,
+                },
+                options: {
+                    name: '_id_index',
+                    unique: true,
+                }
+            }],
+            db_indexes || []);
```

Comment:
I would replace concat with `[item, ...arr]`
```suggestion
        const id_index = {
            fields: {
                _id: 1,
            },
            options: {
                name: '_id_index',
                unique: true,
            }
        };
        this.db_indexes = [id_index,  ...(db_indexes || [])];
```